core/src/main/java/com/microsoft/applicationinsights/internal/perfcounter/JniPCConnector.java

```diff
@@ -68,22 +67,23 @@
      * @return True on success.
      */
     public static boolean initialize() {
+        boolean initializationResult = false;
+
         try {
             if (!SystemInformation.INSTANCE.isWindows()) {
                 InternalLogger.INSTANCE.error("Jni connector is only used on Windows OS.");
                 return false;
             }
 
-            if (loadLibrary()) {
-                return initNativeCode();
-            } else {
-                InternalLogger.INSTANCE.error("Failed to load native dll, Windows performance counters will not be used.");
-            }
+            loadLibrary();
+            initializationResult = initNativeCode();
         } catch (Throwable e) {
-            InternalLogger.INSTANCE.error("Failed to load native dll, Windows performance counters will not be used: '%s'.", e.getMessage());
+            InternalLogger.INSTANCE.error(
+                    "Failed to load native dll, Windows performance counters will not be used. " +
```

Comment:
Alignment - first line (`"Failed to load...') should be 4 chars after `InternalLogger...`.
Second line `"Please make sure...` should be aligned with the first line.